apps/web/src/components/display/AddRecordModal.tsx

```diff
@@ -81,7 +81,7 @@ export const AddRecordModal: React.FC<AddRecordProps> = (props: AddRecordProps)
     ha_pcn: `${job?.ha_pcn?.id || ''}`,
     job_id: `${job?.job_id || ''}`,
     job_title: `${job?.job_title?.id || ''}`,
-    job_location: `${job?.job_location?.id || ''}`,
+    job_location: `${job?.job_location?.length ? job?.job_location[0].id : ''}`,
```

Comment:
@sunghwan-park-fw 
I have selected the Zeroth location from an array of locations to pass the test.
Please implement the multi-select component for job_location.

You can approve this PR and start a new PR to implement multiple job-location select component (in place of select/dropdown).
Or You can continue working in this PR itself.